SPEC/PUBSUB.md

```diff
@@ -78,6 +80,25 @@ ipfs.pubsub.subscribe(topic, receiveMsg, (err) => {
 })
 ```
 
+Or removing all listeners: 
+```JavaScript
+const topic = 'fruit-of-the-day'
+const receiveMsg = (msg) => console.log(msg.toString())
+
+ipfs.pubsub.subscribe(topic, receiveMsg, (err) => {
+  if (err) {
+    return console.error(`failed to subscribe to ${topic}`, err)
+  }
+
+  console.log(`subscribed to ${topic}`)
+
+  setTimeout(() => {
+    // Will unsubscribe ALL handlers for the given topic
+    ipfs.pubsub.unsubscribe(topic);
+  }, 1000)
```

Comment:
Lets use `async/await` in this example, we shouldn't encourage people to use `setTimeout`.

Reply:
This is not required or something I was just following up on [previous example](https://github.com/ipfs/interface-js-ipfs-core/blob/master/SPEC/PUBSUB.md#pubsubunsubscribe), 
anyway I have changed it because [subscribe](https://github.com/ipfs/interface-js-ipfs-core/blob/master/SPEC/PUBSUB.md#ipfspubsubunsubscribetopic-handler-callback) returns a Promise as you mentioned. 
I specifically didn't use `async/await` but instead `.then()` because JS requires using `await` inside `async` method keyword so IDK didn't feel right, let me know if you want it to be changed.